### DISCUSSION TITLE
import python modules and methods

### DISCUSSION THREAD

**Discussion by pterusgithub** [WannesMalfait/Blender-Add-ons#25] [Q&A] [answered]
Could it be possible to import some python modules? For example (certainly not the best one ...),  import the math module in order to use something like math.radian() ?

**Comment by WannesMalfait** [comment, marked as answer]
You can access some basic python commands using `#`. This will evaluate the python expression and tries to read the output as a float. The `math` module is imported internally as a wildcard. So you can use any of the functions from there. Other modules are not available. Here is some more info:

https://github.com/WannesMalfait/Blender-Add-ons/wiki/Users#python-expressions

If you have a good use case for other modules to be available, it is not too difficult to add those. The relevant code is here:
https://github.com/WannesMalfait/Blender-Add-ons/blob/260cf99ea46f2ca4beaa8e203fca741c8d0d202e/math_formula/mf_parser.py#L414-L428

**Reply by pterusgithub** [reply]
thank you ! perfect ! 
PS: my mistake ... math.radians() not math.radian()
